ci/unit/test_main_node.sh

```diff
@@ -24,7 +24,19 @@ sudo ln -s "$SL_PREFIX/include"  /usr/share/spdk/include
 
 # set CMOCKA envs here
 export CMOCKA_MESSAGE_OUTPUT=xml
-export CMOCKA_XML_FILE="$DAOS_BASE"/test_results/%g.xml
+if [ ${WITH_VALGRIND} == 'disabled']; then
```

Comment:
(lint) The mentioned parser error was in this if expression. [SC1009]
(lint) Couldn't parse this test expression. [SC1073]
(lint) You need a space before the ]. [SC1020]
(lint) Missing space before ]. Fix any mentioned problems and try again. [SC1072]